core/config/v2/docs/core.toml

```diff
@@ -183,6 +185,41 @@ StartTimeout = '15s' # Default
 # ListenIP specifies the IP to bind the HTTP server to
 ListenIP = '0.0.0.0' # Default
 
+[WebServer.LDAP]
+# Optional LDAP config if WebServer.AuthenticationMethod is set to 'ldap'
+# ServerTls defines the option to require the secure ldaps
+ServerTls = true # Default
+# SessionTimeout determines the amount of idle time to elapse before session cookies expire. This signs out GUI users from their sessions.
+SessionTimeout = '15m0s' # Default
+# QueryTimeout defines how long queries should wait before timing out, defined in seconds
+QueryTimeout = '2m0s' # Default
+# BaseUserAttr defines the base attribute used to populate LDAP queries such as "uid=$", default is example
+BaseUserAttr = 'uid' # Default
+# BaseDn defines the base dn search filter to apply to every LDAP query, default is example
+BaseDn = 'dc=custom,dc=example,dc=com'
```

Comment:
What is `dn`?

Reply:
Added comments with more context and updated these ones to be more specific about LDAP concepts 🙂 

I will also be updating the docs repo for an LDAP page under admin/authconfig